### DISCUSSION TITLE
Cannot see my mounted drive in Jellyfin

### DISCUSSION THREAD

**Discussion by Foushekis** [tteck/Proxmox#2570] [Help Needed] [answered]
Hi,
I have mounted my Movies folder stored into my nas into my LXC by editing the conf file and putting in the below line:
mp0: /mnt/lxc_shares/nas_rwx/,mp=/mnt/nas
While logged in as root in my LXC I can go into /mnt/nas and I can indeed see the contents of my NAS drive, however when I am trying to add the /mnt/nas folder into jellyfin it comes back blanc. Any ideas?

Thanks

**Comment by Outsmart2383** [comment]
Did you solve this? I'm having the same issue.

I've succesfully mounted my media to my Proxmox host, and bound that mount to my container. I know this is working because I can cd to the directory on the container and see my files/folders. However, when I try to configure the usage of this folder in Jellyfin, it say's the path does not exist.

Full list of steps I have taken: 

On the LXC:
`groupadd -g 10000 <INSERTUSERNAMEOFCHOICE>`

On the Proxmox host:
`mkdir -p /mnt/<INSERTNAMEFORDIRECTORY>`

`{ echo '' ; echo '# Mount CIFS share on demand with rwx permissions for use in LXCs ' ; echo '//NAS-IP-ADDRESS/FOLDERPATH/ /mnt/MOUNTDIRECTORYSPECIFIEDABOVE cifs _netdev,x-systemd.automount,noatime,uid=100000,gid=110000,dir_mode=0770,file_mode=0770,user=smb_username,pass=smb_password 0 0' ; } | tee -a /etc/fstab`

`mount /mnt/<DIRECTORYCREATEDEARLIER>`

`systemctl daemon-reload`

On the LXC:
`{ echo 'mp0: /mnt/<DIRECTORYCREATEDEARLIER>/,mp=/<THEPATHONTHECONTAINERYOUWANTTOBINDTHEMOUNTTO>' ; } | tee -a /etc/pve/lxc/<IDNUMBEROFCONTAINER>.conf`

Clearly all of the above works, because I can see my files in the mount in FileBrowser. This is the same implementation I use for other LXCs (e.g. Paperless) so I don't know why it's not working.

**Reply by Outsmart2383** [reply, marked as answer]
Alright, so I've solved it for me.

For some reason, creating the user on the Proxmox host with the ID 10000 didn't work.

Manually changing the mount uid and gid to the ID of Jellyfin did. I got this by running `ps aux | grep jellyfin` on the Jellyfin LXC.

**Comment by Foushekis** [comment]
Boom! You are correct. Thanks very much mate
